src/digraph.rs

```diff
@@ -1275,15 +1281,15 @@ impl PyDiGraph {
         Ok(out_index)
     }
 
-    /// Add new edges to the dag.
+    /// Add new edges to the graph.
```

Comment:
Nice find! Originally `PyDiGraph` was only for DAGs. `PyDAG` took that role later